### ISSUE TITLE
Check landmark regions in theme

### ISSUE THREAD

**Issue by robgasc** [ATLAS-Illinois/Illinois-Framework-Theme#1187]
**Describe the bug**
Landmarks are an important mechanism for conveying the page layout and allowing users to bypass sections they are not interested in.
a. All page content needs to be inside a Landmark region. Several content areas are outside Landmarks which will hamper screen reader users trying to navigate to specific areas of the page. 
b. Each page must have one of the following: HEADER/BANNER, MAIN, FOOTER/CONTENTINFO and NAVIGATION as needed. Landmarks are missing on several pages throughout the site.
(Reference WCAG 2.1 - 1.3.1 Info and Relationships (A) - https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-
programmatic.html, 2.4.1 Bypass Blocks (A) - https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html

**Expected behavior**
All content is inside a landmark.

**Comment by trubach** [comment]
To the best of our knowledge, this issue is resolved. The scanner used doesn't recognize the regions in the shadow DOM and so it is not an issue.
